src/components/UserDetails/UserDetails.js

```diff
@@ -1,12 +1,25 @@
 import React from 'react';
 import './UserDetails.css';
+import profilePic from '../../Images/user.png';
+import homeIcon from '../../Images/home.png';
+import veggieIcon from '../../Images/carrot.png';
 
-const UserDetails = () => {
+const UserDetails = ({user}) => {
```

Comment:
just noting for consistency - I appreciate and also tend to follow this pattern for destructuring, even when only a single prop is passed